src/memory/postgresql_store/memory.rs

```diff
@@ -74,6 +74,49 @@ impl<const ADDRESS_LENGTH: usize, const WORD_LENGTH: usize>
             _marker: PhantomData,
         })
     }
+
+    // Initialize the table in the database, used only for benching
+    #[cfg(feature = "test-utils")]
+    pub async fn connect_and_init_table(
+        db_url: String,
+        table_name: String,
+    ) -> Result<Self, PostgresMemoryError> {
+        use deadpool_postgres::Config;
+        use tokio_postgres::NoTls;
+
+        let mut pg_config = Config::new();
+        pg_config.url = Some(db_url.to_string());
+        let test_pool = pg_config.builder(NoTls)?.build()?;
+
+        let m = Self::connect_with_pool(test_pool, table_name.to_string()).await?;
+
+        m.initialize_table(db_url.to_string(), table_name, NoTls)
+            .await?;
+        Ok(m)
+    }
+
+    /// Deletes all rows from a given table, used only for benches
+    #[cfg(feature = "test-utils")]
+    pub async fn clear(&self) -> Result<(), PostgresMemoryError> {
+        let conn = self.pool.get().await?;
+        // Drop the existing table
+        conn.execute(&format!("DROP TABLE IF EXISTS {};", self.table_name), &[])
+            .await?;
+
+        // Recreate the table
+        conn.execute(
+            &format!(
+                "CREATE TABLE IF NOT EXISTS {} (
+                    a BYTEA PRIMARY KEY CHECK (octet_length(a) = {}),
+                    w BYTEA NOT NULL CHECK (octet_length(w) = {})
+                );",
+                self.table_name, ADDRESS_LENGTH, WORD_LENGTH
+            ),
+            &[],
+        )
+        .await?;
+        Ok(())
```

Comment:
No, I am almost sure there is a way in SQL for atomically dropping everything in a table.